### ISSUE TITLE
[Feature Request] Add Black code formatting and tooling

### ISSUE THREAD

**Issue by apoclyps** [ajurna/cbwebreader#4]
### Request

I would like to add [Black](https://pypi.org/project/black/) to the project as the Python code formatter because Black makes code review faster by producing the smallest diffs possible and ensuring consistent best practices across the entire codebase.

Benefits of black:

* Black gives you speed, determinism, and freedom from pycodestyle nagging about formatting. You will save time and mental energy for more important matters.
* Blackened code looks the same regardless of the project you're reading. Formatting becomes transparent after a while and you can focus on the content instead.
* Other benefits of black: https://www.mattlayman.com/blog/2018/python-code-black/

### Proposal

Reformat the existing codebase as black as a single atomic commit. This will ensure the codebase aligns to Black conventions prior to adding tooling to enforce this for future additions.

Once black has been applied to the code base in https://github.com/ajurna/cbwebreader/pull/5 and the PR has been merged, a follow-up PR would be opened to add black tooling to the codebase. This could consist of:

* A readme detailing how to use black
* Addition of [pytest-black](https://pypi.org/project/pytest-black/) to allow pytest to flag any infractions from the standard. This becomes incredibly useful when combined with continuous integration.

**Comment by ajurna** [comment]
I prefer a slightly longer line length (ala 120 or so) but otherwise i try to keep to pep8. So i don't mind this.

**Comment by apoclyps** [comment]
I've updated the initial PR (#5 Apply Black formatting) to a line length of 119 characters. As a hard limit, it's recommended to keep all lines less than 119 characters (which is the width of GitHub code review).

I'll follow up with a PR to add pytest-black once #3 lands, as it adds some of the initial setup required for what I detailed above..

**Comment by ajurna** [comment]
this is now merged.
